certified-connectors/Data247logic/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3474 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Data247",
+    "description": "Welcome to the Data247 API! You can use our API to access the various data services we provide.",
+    "version": "1.0",
+    "contact":{
+		"name":"Marc Orenberg",
+		"url":"https://www.data247.com/docs/#introduction",
+		"email":"mohammad@logictree.com"
+	}
+  },
+  "host": "api.data247.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3.0": {},
+    "/v3.0/MT/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "response": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "status"
+                    },
+                    "results": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "phone": {
+                            "type": "string",
+                            "description": "phone"
+                          },
+                          "wless": {
+                            "type": "string",
+                            "description": "wless"
+                          },
+                          "carrier_name": {
+                            "type": "string",
+                            "description": "carrier_name"
+                          },
+                          "carrier_id": {
+                            "type": "string",
+                            "description": "carrier_id"
+                          },
+                          "sms_address": {
+                            "type": "string",
+                            "description": "sms_address"
+                          },
+                          "mms_address": {
+                            "type": "string",
+                            "description": "mms_address"
+                          }
+                        }
+                      },
+                      "description": "results"
+                    }
+                  },
+                  "description": "response"
+                }
+              }
+            }
+          }
+        },
+        "summary": "MESSAGING SERVICES Text@",
```

Comment:
@Data247logic Could you please let us know why the summary value is capitalized ? Please provide ALL the summary values in small letters (with first letter as capital letter).
And what does mean by "Text@" ? If it is not a brand or product name, please make it "text" instead of "Text@"

Reply:
this is also fixed, please review.